tsl/test/t/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ set(PROVE_TEST_FILES 001_job_crash_log.pl 002_logrepl_decomp_marker.pl)
 
 set(PROVE_DEBUG_TEST_FILES 003_mvcc_cagg.pl)
 
+if((${PG_VERSION_MAJOR} GREATER_EQUAL "16"))
```

Comment:
Whats the reason this is PG16 and above? Let's add a comment here if its necessary.